Integrations/integration-IPInfo_PY.yml

```diff
@@ -0,0 +1,237 @@
+description: Use the ipinfo.io API to get data about an IP address
+configuration:
+- display: API Token (optional)
+  name: token
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |+
+    import requests
+    import os
+    import json
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    # ''' GLOBAL VARS '''
+    BASE_URL = 'http://ipinfo.io/'
+    USE_SSL = not demisto.params().get('insecure', False)
+    APIKEY = demisto.params()['apikey']
+    IP = demisto.args().get('ip')
+    FIELD = demisto.args().get('field')
+
+    headers = {
+        'Content-Type': "application/json",
+        'Accept': "application/json"
+        }
+
+    ## Helper Functions
+
+    ## Sends Request to IP Info
+    def send_request():
+        if APIKEY is None:
+            r = requests.request("GET",BASE_URL+IP+'/json',headers=headers)
+        else:
+            r = requests.request("GET",BASE_URL+IP+'/json?token='+APIKEY,headers=headers)
+        data = r.json()
+        return data
+
+    # Checks to see if Request returned an error message
+    def error_check(data):
+        if 'error' in data:
+            errormsg = data['error']['message']
+            md = '## There has been an error: '+errormsg+''
```

Comment:
why do you append the empty string at the end?
use `format` for string formatting instead of concatenating with `+`